### ISSUE TITLE
[Bug]: AttributeError when using `.. include::`

### ISSUE THREAD

**Issue by htgoebel** [rstcheck/rstcheck#212]
### To Dos

- [X] I tested with the [latest released version](https://github.com/rstcheck/rstcheck/releases/latest).

- [X] I have checked the [issues](https://github.com/rstcheck/rstcheck/issues) and think that this is not a duplicate.

- [X] I added a very descriptive title to this issue.

### Example Code (rst)

```rst
This is the outer text.

.. include:: _common_definitions.txt
```

### Relevant log output

```Shell
WARNING:rstcheck_core.checker:An `AttributeError` error occured. This is most probably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'index.rst'. The reason can also be another directive. For more information see the FAQ (https://rstcheck-core.rtfd.io/en/latest/faq) or the corresponding github issue: https://github.com/rstcheck/rstcheck-core/issues/3.
```

### Description

The file above is named `index.rst`. There is a second file (`_common_definitions.txt`):
```
This is some included text.
```

- running `rst2html5.py index.rst` works as expected: the output html contains the included text.
- running `rstcheck index.rst` gives the warning shown above.

### Operating System

Linux

### Python Version

3.10

### rstcheck Version

rstcheck CLI Version: 6.2.0 rstcheck-core Version: 1.2.0
